### ISSUE TITLE
Possibility to add a new table or more

### ISSUE THREAD

**Issue by sikelio** [edisonneza/jspdf-invoice-template#31]
I think it could be a good idea to give the possibility to add multiple table. For example some information's cannot have the same column designation name.

**Comment by edisonneza** [comment]
Yes right, it's something I've been thinking about. Added to my to-do list.
